File: js-version/src/builder/EmployeeBuilder.js
import {Employee} from "./Employee.js";

export class EmployeeBuilder {
   employee = new Employee()
   
   setEmployeeFirstName(firstName) {
	  this.employee.firstName = firstName
	  return this;
   }
   
   setEmployeeLastName(lastName) {
	  this.employee.lastName = lastName
	  return this;
   }
   
   setEmployeeAge(age) {
	  this.employee.age = age
	  return this;
   }
   
   build() {
	  return this.employee;
   }
}

// module.exports = {
//    EmployeeBuilder
// }

File: python-version/src/builder/EmployeeBuilder.py
from src.builder.Employee import Employee


class EmployeeBuilder:
    employee = Employee()

    def set_employee_first_name(self, first_name):
        self.employee.firstName = first_name
        return self

    def set_employee_last_ame(self, last_name):
        self.employee.lastName = last_name
        return self

    def set_employee_age(self, age):
        self.employee.age = age
        return self

    def build(self):
        return self.employee
